patches/rtorrent/rtorrent-ml-fixes.patch

```diff
@@ -0,0 +1,113 @@
+From: stickz <stickman002@mail.com>
+Date: Wed, 4 Oct 2023 07:12:02 -0400
+Subject: [PATCH] Fix various memory leaks
+
+Resolves a potential memory leak with the curses UI when filtering torrents.
+
+Resolves a memory leak with dynamic commands in the .rtorrent.rc file.
+
+Resolves a memory leak during software initialization with choke groups.
```

Comment:
IIUC this seems to be three specific patches. Could you split them in a respective patch file?

Reply:
Alrighty, patch files are separated.